Sources/TGCardViewController/cards/TGTableCard.swift

```diff
@@ -155,8 +155,14 @@ open class TGTableCard: TGCard {
         return
     }
     
-    embeddedScrollView.contentInset.bottom = scrollCardView.safeAreaInsets.bottom
-    embeddedScrollView.verticalScrollIndicatorInsets.bottom = scrollCardView.safeAreaInsets.bottom
+    if scrollCardView.safeAreaInsets.bottom == 0 {
+      embeddedScrollView.contentInset.bottom = scrollCardView.safeAreaInsets.bottom + 16
```

Comment:
Can you add a comment where the 16 comes from and why it’s needed? Is it just the standard padding to not have something stuck to the bottom, or is it (partially) hidden otherwise?

Reply:
Done